test/test_mm.py

```diff
@@ -0,0 +1,283 @@
+"""
+Test bmm operation on tensors with various layouts.
+"""
+
+import torch
```

Comment:
testing changes look OK. @cpuhrsch is probably the best principal reviewer for this, but I'm not sure we want to add a new test file, especially not one with such a generic name